src/main/java/me/ironexception/universalfriends/configuration/Operations.java

```diff
@@ -20,6 +21,32 @@
         return filterMatchingPersons(configuration, t -> Association.closestToValue(t.getValue()) == association);
     }
 
+    /**
+     * Gives you a set of all your friends. No matter how friendly they are.
+     * The difference to {@link Operations#getByCloseToAssociation} is that you also get players with the
+     * friendliness value between excluding 0 and 0.5.
+     *
+     * @param configuration the config.
+     * @param <T>
+     * @return all players that are saved as your friends. No matter how friendly they are.
+     */
+    public static <T extends IPerson> Set<T> getAllFriends(final Configuration<T> configuration) {
+        return getInFriendlinessRange(configuration, Standard.STANDARD_NEUTRAL, Standard.STANDARD_MAX_FRIENDLINESS);
+    }
+
+    /**
+     * Gives you a set of all your enemies. No matter how hostile they are.
+     * The difference to {@link Operations#getByCloseToAssociation} is that you also get players with the
+     * friendliness value between -0.5 and excluding 0.
+     *
+     * @param configuration the config.
+     * @param <T>
+     * @return all players that are saved as your enemy. No matter how hostile they are.
+     */
+    public static <T extends IPerson> Set<T> getAllEnemies(final Configuration<T> configuration) {
+        return getInFriendlinessRange(configuration, Standard.STANDARD_MIN_FRIENDLINESS, Standard.STANDARD_NEUTRAL);
```

Comment:
Ditto

Reply:
you mean this is the same issue as with getAllFriends?

then `Ditto` also from me